WordPress/src/main/res/layout/category_detail_activity.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <fragment
```

Comment:
Nitpick: We might want to replace the `<fragment>` tag with `FragmentContainerView` as suggested in Android Studio.

Reply:
@ashiagr Yup. You are right. Didn't know about `FragmentContainerView` until now. 😅 . Done in 257b8ff